src/ImageGen.py

```diff
@@ -64,8 +64,10 @@ def get_images(self, prompt: str) -> list:
 
         # Use regex to search for src=""
         image_links = regex.findall(r'src="([^"]+)"', response.text)
+        # Remove size limit
+        normal_image_links = [link.split("?w=")[0] for link in image_links]
```

Comment:
Can we make this optional?

Reply:
Yes, we can make this an option, it's a good idea.